include/xen/arm/interface.h

```diff
@@ -15,7 +15,7 @@
 #define uint64_aligned_t u64 __attribute__((aligned(8)))
 
 #define __DEFINE_GUEST_HANDLE(name, type) \
-	typedef struct { union { type *p; uint64_aligned_t q; }; }  \
+	typedef struct { union { type * p; uint64_aligned_t q; }; }  \
```

Comment:
Is this really needed?

Reply:
-:44: CHECK:SPACING: spaces preferred around that '*' (ctx:WxV)
#44: FILE: include/xen/arm/interface.h:18:
+       typedef struct { union { type *p; uint64_aligned_t q; }; }  \